README.md

```diff
@@ -81,6 +86,20 @@ $ driftctl scan --from tfstate://terraform.tfstate
 $ driftctl scan --from tfstate+s3://my-bucket/path/to/state.tfstate
 ```
 
+Or with ou docker image:
```

Comment:
It could be nice to move this paragraph before manual installation step 

![image](https://user-images.githubusercontent.com/6154987/102089245-40142300-3e1c-11eb-83d4-c41203a0d6f4.png)

And put a title like manual : `#### Docker`

Reply:
:heavy_check_mark: